CMakeLists.txt

```diff
@@ -74,6 +77,7 @@ if(BUILD_TESTING)
   include(Catch)
 endif()
 
+
```

Comment:
Go through the diff, either here on github in "Files changed" tab or using `git diff` and remove these unrelated changes. This space for instace, blank comments later, etc. make the PR harder to review than it needs to be.